### PR TITLE
fix #8682: Add new targeting for first run non-RTAMO users

### LINKED ISSUE #8682: Add new targeting for first run non-RTAMO users
Exclude RTAMO users from enrollment targeting by checking attributionData.source !== ‘addons.mozilla.org’

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-3380)

### PR DESCRIPTION
**Pull request by AllegroFox**
Because

This targeting will be used in the Window vs Tab modal with Easy Setup screen experiment, intended for Fx112

This commit

Duplicates the `FIRST_RUN_NEW_PROFILE_HAS_PIN_NEED_DEFAULT_WINDOWS_1903` targeting, with an added check for `attributionData.source !== ‘addons.mozilla.org’`

### CODE REVIEW

File: experimenter/experimenter/targeting/constants.py [hunk 1]
```diff
@@ -180,6 +180,35 @@ def __post_init__(self):
...
+    slug="first_run_new_profile_exclude_rtamo",
+    description=(
+        "First start-up users (e.g. for about:welcome) on Windows 1903+, "
+        "with a new profile, needing default w/ pin, excluding RTAMO"
+    ),
+    targeting=(
+        "{first_run} && {has_pin} && {exclude_rtamo}".format(
+            first_run=FIRST_RUN_NEW_PROFILE_NEED_DEFAULT_WINDOWS_1903.targeting,
+            has_pin=HAS_PIN,
+            exclude_rtamo="attributionData.source !== ‘addons.mozilla.org’",
```
Comment (punamdahiya) [inline experimenter/experimenter/targeting/constants.py:197]: I believe quotes above are different from whats used here https://github.com/mozilla/experimenter/blob/e3620a81b4f6bdbe07c3c19436a5fa0bea039b7b/experimenter/experimenter/targeting/constants.py#L78

Comment (AllegroFox) [reply experimenter/experimenter/targeting/constants.py:197]: ooh yep, good catch!

**Review by jaredlockhart — APPROVED** [review]
LGTM 👍

**Comment by jaredlockhart** [comment]
@AllegroFox I fixed the formatting, also in JEXL it's `!=` instead of `!==`.  You wanna take a quick look before I merge?

File: experimenter/experimenter/targeting/constants.py [hunk 2]
```diff
@@ -180,6 +180,35 @@ def __post_init__(self):
...
+            first_run=FIRST_RUN_NEW_PROFILE_NEED_DEFAULT_WINDOWS_1903.targeting,
+            has_pin=HAS_PIN,
+            attribution="attributionData.source != 'addons.mozilla.org'",
+        )
+    ),
+    desktop_telemetry=(
+        "{first_run} AND {has_pin} AND {attribution}".format(
+            first_run=FIRST_RUN_NEW_PROFILE_NEED_DEFAULT_WINDOWS_1903.desktop_telemetry,
+            has_pin=HAS_PIN,
+            attribution="attributionData.source != 'addons.mozilla.org'",
```
Comment (Mardak) [inline experimenter/experimenter/targeting/constants.py:204]: I don't think this is actually used, but it's confusing that you've mixed targeting syntax with telemetry syntax. https://github.com/mozilla/experimenter/blob/d6a782f7e8b2b55c4eebb96d9b33cd80b09b2f81/experimenter/experimenter/targeting/constants.py#L81-L83

**Pull request merged**